src/scraper/spiders/el_pitazo.py

```diff
@@ -0,0 +1,73 @@
+# Internal imports
+import scraper.utils as utils
+
+# External imports
+import scrapy
+
+# Python imports
+from typing import List
+
+
+class ElPitazoSpider(scrapy.Spider):
+    """
+        Spider to scrape ElPitazo data 
+    """
+
+    name = "el_pitazo"
+
+    start_urls = []
+
+    def parse(self, response):
+        """
+            Returns a dict like structure with the following 
+            fields:
+                + title
+                + date
+                + categories
+                + body
+                + author 
+                + tags
+        """
+
+        # These are simple properties, just get its text with a valid
+        # selector
+        title = utils.get_element_text(".tdb-title-text", response) or ""
+        date = utils.get_element_text(".entry-date", response) or ""
+        author = utils.get_element_text(".tdb-author-name", response) or ""
+
+        body = self._get_body(response)
+
+        tags = self._get_tags(response)
+
+        # categories
+        categories = response.css(".tdb-entry-category").getall()
+        categories = list(map(utils.strip_http_tags, categories))
+
+        return {
```

Comment:
This is related to my previous comment about using typed objects for outputs. Or more precisely, a hierarchy of DataClasses.